core/client/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -260,12 +283,12 @@ private long getNextBlockId() throws IOException {
     try (CloseableResource<FileSystemMasterClient> masterClient = mContext
         .acquireMasterClientResource()) {
       return masterClient.get().getNewBlockIdForFile(mUri);
-    } catch (AlluxioException e) {
+    } catch (AlluxioStatusException e) {
       throw new IOException(e);
```

Comment:
this should either always be `e.toIOException()`, or we should not have that method